recipes/pcre2/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pcre2" %}
+{% set version = "10.30" %}
+{% set sha256 = "b549873a39f804480c2e6145a78adcba53e38162d90ef6ea92384f6ecf2fde76" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: ftp://ftp.csx.cam.ac.uk/pub/software/programming/pcre/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [osx]
```

Comment:
Also Windows should have been disabled here as there was nothing being done for that platform. Added PR ( https://github.com/conda-forge/pcre2-feedstock/pull/1 ) to the feedstock to disable Windows builds (for now).